src/Styleguide/Elements/Flex.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components"
+import { FlexboxProperties, MarginProperties } from "../StyledSystemTypes"
```

Comment:
Since its somewhat conventional to see `SomethingProps` around our TS codebases, can we shortent `Properties` a bit?

Reply:
Yeah, I can do that. I was trying to distinguish between react "props" and CSS properties.